PetrolEngine.cpp

```diff
@@ -5,16 +5,16 @@ PetrolEngine::PetrolEngine(int power, float capacity, int gears)
     : power_(power)
     , capacity_(capacity)
     , gears_(gears)
-    , currentGear_(0)
+    , currentGear_(1)
 {
     std::cout << __FUNCTION__ << std::endl;
 }
 
 void PetrolEngine::changeGear(int gear)
 {
-    // TODO: Add checking if gear is between -1 and gears_
-    // -1 is for REAR
-    // 0 is for NEUTRAL
+    if(gear < -1 || gear > gears_ || (currentGear_ != 0 && gear == -1)) {
```

Comment:
I suggest you to avoid magic numbers and define variables like -1 as constexpr or const variables

Reply:
Agree